tf2onnx/tf_loader.py

```diff
@@ -29,6 +29,7 @@ def is_tf2():
     from tensorflow.python.framework import convert_to_constants, func_graph, function_def_to_graph
 else:
     from tensorflow.python.framework.graph_util import convert_variables_to_constants
+    from tensorflow.python.framework import function_def_to_graph
```

Comment:
Seems ok but I'm worried this might not be there in super old tensorflow versions. In theory we don't support them anymore but in practice we see people still using things like tf-1.4.
Could we move this import a few lines lower into the 
elif LooseVersion(tf.__version__) >= "1.13"
block?

Reply:
Well sure, but then you get the same very uninformative exception in tf 1.12 (and the next person will have to redo my debugging :/), and support is claimed for "tf-1.12 and up".

I think it is a reasonable principle that references to variables that do not exist (NameError) should never happen - it is a bug in your code. The problem is that the responsibility for handling various tf-version is unstructured and distributed across modules: callers of `tf_loader` will have to know that some functions implemented in that module are only usable depending on tf version etc. I updated the PR with a suggestion that will at least give an informative error when this happens.

Reply:
I agree, The support for multiple versions is not optimal. We tried to isolate it in tf_loader, tf_util  and unit tests (later is a real royal pain) but forgot that functions can show up before tf2.
Adding a more formal interface to handle multiple version: there are still some issues with tf2 we are working on and once that is done we'll find some time to restructure tf_loader to make this cleaner.